en/docs/reference/product-apis/service-catalog-apis/service-catalog-v1/service-catalog-v1.yaml

```diff
@@ -0,0 +1,1089 @@
+          type: string
+          example: http://localhost/pizzashack
+        definitionType:
+          type: string
+          description: The type of the provided API definition
+          example: OAS3
+          enum:
+            - OAS2
+            - OAS3
+            - WSDL1
+            - WSDL2
+            - GRAPHQL_SDL
+            - ASYNC_API
+        securityType:
+          type: string
+          description: The security type of the endpoint
+          example: BASIC
+          enum:
+            - BASIC
+            - DIGEST
+            - OAUTH2
+            - X509
+            - API_KEY
+            - NONE
+          default: NONE
+        mutualSSLEnabled:
+          type: boolean
+          description: Whether Mutual SSL is enabled for the endpoint
+          default: false
+          example: false
+        usage:
+          type: integer
+          description: Number of usages of the service in APIs
+          readOnly: true
+          example: 1
+        createdTime:
+          type: string
+          readOnly: true
+          example: '2020-02-20T13:57:16.229'
+        lastUpdatedTime:
+          type: string
+          readOnly: true
+          example: '2020-02-20T13:57:16.229'
+        md5:
+          type: string
+          example: 36583a6a249b410e7fc4f892029709cac09763ddb230e1a829d5f9134d1abd07
+        definitionUrl:
+          type: string
+          example: https://petstore.swagger.io/v2/swagger.json
+    
+    ServiceInfo:
+      type: object
+      title: The Service Info object
+      properties:
+        id:
+          type: string
+          example: 01234567-0123-0123-0123-012345678901
+          readOnly: true
+        name:
+          type: string
+          pattern: '^[^\*]+$'
+          example: PizzashackEndpoint
+        key:
+          type: string
+          example: PizzashackEndpoint-1.0.0
+        version:
+          type: string
+          example: v1
+        md5:
+          type: string
+          example: 36583a6a249b410e7fc4f892029709cac09763ddb230e1a829d5f9134d1abd07
+          
+    #-----------------------------------------------------
+    # The Service Schema resource
+    #-----------------------------------------------------
+    ServiceSchema:
+      type: object
+      title: The Service Schema object
+      required:
+        - serviceMetadata
+        - definitionFile
+      properties:
+        serviceMetadata:
+          $ref: '#/components/schemas/Service'
+        definitionFile:
+          type: string
+          format: binary
+        inlineContent:
+          type: string
+          description: Inline content of the document
+
+    #-----------------------------------------------------
+    # The settings resource
+    #-----------------------------------------------------
+    Settings:
+      title: SettingsDTO
+      properties:
+        scopes:
+          type: array
+          items:
+            type: string
+            example: service_catalog:service_view
```

Comment:
shall we add all the supported scopes into the example?

Reply:
Fixed in 00f53c7